demystifying_machine_learning/demystifying_machine_learning.md

```diff
@@ -512,8 +512,8 @@ Here are a few of our favorites:
 
 - [Google's Intro to Machine Learning](https://developers.google.com/machine-learning/intro-to-ml)
 - [DeepLearning.ai](https://www.deeplearning.ai/): Several world-class courses on machine learning topics ranging from intro to very advanced
-- [Introduction to Statistical Learning at Stanford](https://www.statlearning.com/online-course) (there is also a free [text book](https://hastie.su.domains/ISLR2/ISLRv2_website.pdf) and [R package](https://cran.r-project.org/web/packages/ISLR2/ISLR2.pdf) that go with this course)
-- [Introduction to Applied Machine Learning at the University of Wisconsin](https://dionysus.psych.wisc.edu/iaml/)
+- [Introduction to Statistical Learning at Stanford](https://www.statlearning.com/online-courses) (there is also a free [text book](https://hastie.su.domains/ISLR2/ISLRv2_corrected_June_2023.pdf.download.html) and [R package](https://cran.r-project.org/web/packages/ISLR2/ISLR2.pdf) that go with this course)
```

Comment:
Same as the previous comment, would be nice if this wasn't a download link (though it might not be possible).

Reply:
Unfortunately couldn't find this :(